stat/stat_test.go

```diff
@@ -1625,5 +1626,75 @@ func TestStdScore(t *testing.T) {
 			t.Errorf("StdScore mismatch case %d. Expected %v, Found %v", i, test.z, z)
 		}
 	}
+}
+
+func ExampleQuantile_median() {
```

Comment:
As said above, I think we should take this out until we have the linear piecewise approximation.